packages/helper-grammar-regex-collection/index.js

```diff
@@ -11,7 +11,7 @@ const importMembers = regex`[\r\n\s\w{},*\$]*`;
 const from = regex`\s from \s`;
 
 const REQUIRE = regex`
-  ( require(\.resolve)? | import )
+  ( require(\.resolve)? | import(\.resolve)? )
```

Comment:
As far as I can tell from the [spec] linked in #315, `import.resolve` doesn't exist. The corresponding tests should be removed instead.

[spec]: https://github.com/tc39/proposal-dynamic-import